org.eclipse.xtext/src/org/eclipse/xtext/workspace/IOutputFolder.java

```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2015 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.workspace;
+
+/**
+ * Describes directories that contain output files of a project. 
+ */
+public interface IOutputFolder extends ISourceFolder {
```

Comment:
what is the idea behind a output folder beeing a source folder

Reply:
something like javasourcelanguage?

Reply:
Just to reuse all methods. Like noted above, I am not that satisfied with this solution... 

Maybe instead there should be a super interface of both `ISourceFolder` and `IOutputFolder`, and this super interface would declare all methods etc (currently in `ISourceFolder`), and these sub interfaces would be empty.